### ISSUE TITLE
Rrevent creating new annotations if user isn't logged in

### ISSUE THREAD

**Issue by balmas** [perseids-project/digital_milliet#55]
it would be nice if we could hide or remove the ability for users to create new annotations on images in the mirador viewer if they aren't logged in.  It seems the ability edit or delete existing annotations is alreayd triggered by that, but the new annotation features are still there. saving doesn't do anything but it also doesn't warn the user that it won't.

**Comment by PonteIneptique** [comment]
Yeah I saw that but could not find (or understand) where to modify this behaviour...
